packages/terra-content-container/src/terra-dev-site/doc/example/ContentContainerScrollRef.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import ContentContainer from 'terra-content-container/lib/ContentContainer';
+import classNames from 'classnames/bind';
+
+import styles from './ContentContainerDocCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const containerRef = React.createRef();
```

Comment:
This would throw some warnings in the console as scrollRefCallback expects function to be passed. as this will pass an object.

Reply:
With this example we cannot differentiate with fill container example we have currently. we will have to draft this example to show what actually we can do using scrollRefCallback prop for the end user.

Reply:
Yeah, maybe display the scroll position values on the example page. These can be referenced from the ref we return in the `scrollRefCallback` prop.

Reply:
Resolved in 52bf2a9